packages/maskbook/src/plugins/ITO/UI/RegionSelect.tsx

```diff
@@ -0,0 +1,232 @@
+import * as React from 'react'
+import { useState, useRef, useCallback, forwardRef, useImperativeHandle } from 'react'
+import type { InputBaseComponentProps } from '@material-ui/core'
+import { createStyles, makeStyles } from '@material-ui/core/styles'
+import { useDebounce } from 'react-use'
+import {
+    Typography,
+    Popover,
+    List,
+    ListItem,
+    ListItemText,
+    ListItemIcon,
+    Checkbox,
+    FormControl,
+    InputAdornment,
+    FilledInput,
+} from '@material-ui/core'
+
+import { Flags } from '../../../utils/flags'
+import { useI18N } from '../../../utils/i18n-next-ui'
+import { useRegionList } from '../hooks/useRegion'
+import type { RegionCode } from '../hooks/useRegion'
+import { usePortalShadowRoot } from '../../../utils/shadow-root/usePortalShadowRoot'
+
+export interface RegionSelectProps extends InputBaseComponentProps {
+    value: RegionCode[]
+    // onChange: React.ChangeEvent<{ value: RegionCode[] }> // TODO how to implement the ChangeEvent interface?
```

Comment:
Not possible. Cause it represents a native event which does not contain this kind of data

Reply:
the material ui multi select emits an array value on change event,  but I cannot do it like the [source code](https://github.com/mui-org/material-ui/blob/next/packages/material-ui/src/Select/SelectInput.js#L272-L283) and pass the type check